docs/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 All notable changes to this project will be documented in this file.
 
+## [Unreleased] - ././2020
+
+- [🐞 Fix: vertical style issue by @hadialqattan](https://github.com/hakancelik96/unimport/pull/85)
```

Comment:
This URL looks wrong, should point to `86`, not `85`.

Reply:
My bad.